WordPressKitTests/WordPressOrgRestApiTests.swift

```diff
@@ -114,4 +126,10 @@ extension WordPressOrgRestApi {
     }
 }
 
+extension WordPressOrgRestApi.Site {
+    static func dotCom(siteID: UInt64, bearerToken: String) -> Self {
+        .dotCom(siteID: siteID, bearerToken: bearerToken, apiURL: WordPressComRestApi.apiBaseURL)
+    }
+}
```

Comment:
Note for future readers: This method looks unused in the diff, but it's actually a way to bridge the new implementation, which expects an `apiURL` parameter, with the existing code that did not have it.

See for example line 103 in this file.